katago_server/games/migrations/0001_initial.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 2.2.8 on 2019-12-24 00:11
+
+from django.db import migrations, models
+import katago_server.games.models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
```

Comment:
The reason we need a migration already here initially is because we are "migrating" from the "empty model" to the initial model that actually has this stuff?

Reply:
All change to db schema should be a mutation. So you can migrate rollback a change.

Normally you should mutate a existing mutation. I didn't respect this here since the model is changing fast wich generate nasty migration change (eg if you generate "selfplay" model, then create "network" and add the foreign key "network_id" in "selfplay" django will ask you smartly to prpvide default in migration so old "selfplay" can have a network associated. Which is smart when you have a lot of data but more complexe as the start than resetting db and redoibg a clean init migration).